metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -476,6 +477,20 @@ object ServerCommands {
     """|This command should be sent in with the LSP [`TextDocumentPositionParams`](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams)
        |""".stripMargin,
   )
+  final case class ExtractMethodParams(
+      param: TextDocumentIdentifier,
+      range: lsp4j.Range, // from LSP
+      extractPosition: lsp4j.Range,
```

Comment:
```suggestion
      extractPosition: lsp4j.Position,
```
it seems we only need a position?